app/controllers/teams/invitations_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Teams::InvitationsController < TeamsController
+  skip_before_action :find_team, only: [:destroy]
```

Comment:
See comment in routes.

I would rather add an extra scope which is `before_action :check_admin, only: [:create, :destroy]` and then do something like `current_user.managed_teams.where(id: params[:team_id]).exists?` there. It feels more clear of intention from a security POV than nesting that check within the methods.

Reply:
Agree with this.